mmtrack/models/losses/triplet_loss.py

```diff
@@ -0,0 +1,65 @@
+import torch
+import torch.nn as nn
+from mmcls.models import LOSSES
```

Comment:
Use the `LOSSES` in this codebase is better.

Reply:
This codebase do not have `LOSSES`.